### DISCUSSION TITLE
user friendly installation script

### DISCUSSION THREAD

**Discussion by jayvgithub** [netbox-community/netbox#8616] [Ideas]
hi devs,

iam trying to install netbox and found no proper installation  doc or script with one click installation for user friendly it will be helpfull for more users to run.

**Comment by empusas** [comment]
I think that will be not easy as you have to deal with different Linux distributions.
I had to install Netbox on a REL 8 with SE Linux. Basically nothing from the instructions worked and the upgrade does not work either as all the files and directories have wrong owner/permissions after the upgrade, as a result of installing it with sudo.

**Reply by jayvgithub** [reply]
there is  script which is outdated.. can be update from a dev
https://github.com/hdkmike/netbox-scripts

**Comment by mtinberg** [comment]
I'm not sure you can have a "one-click-install" of a 3-tier web application outside of Docker containers or Ansible playbooks, which still require configuration, backups, redundancy etc. to be provided by the sysadmin.  I think if you don't have the technical background, time, or resources to learn how to sysadmin a webapp yourself then paying for someone to host it for you makes the most sense.  While I'm sure NS1 would like the business, there are plenty of small web development shops that have Django experience who could set it up and maintain it for you at an hourly rate.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
